### PR TITLE
Fix type of null handling

### PR DESCRIPTION
**Pull request by aspirisen**

### CODE REVIEW

File: src/guards/isUploadedFile.ts [hunk 1]
```diff
@@ -2,11 +2,11 @@ import { isPlainObject } from 'is-plain-object';
 
 import type { UploadedFile } from '../types';
 
-export function isUploadedFile(file: any | undefined): file is UploadedFile {
+export function isUploadedFile(file: Record<string, any> | undefined | null): file is UploadedFile {
     return (
         typeof file === 'object' &&
         isPlainObject(file) &&
-        typeof file.uuid === 'string' &&
+        typeof file?.uuid === 'string' &&
```
Comment (e1himself) [inline src/guards/isUploadedFile.ts]: > `typeof file?.uuid === 'string'`

Hm... why is this necessary? Isn't `isPlainObject` already checking for `file` object to be not null?

File: src/guards/isUploadedImage.ts [hunk 2]
```diff
@@ -1,14 +1,18 @@
...
 
-export function isUploadedImage(file: any): file is UploadedImage {
+export function isUploadedImage(
+    file: Record<string, any> | undefined | null,
+): file is UploadedImage {
     return (
         typeof file === 'object' &&
-        typeof file.original_height === 'number' &&
+        isPlainObject(file) &&
+        typeof file?.original_height === 'number' &&
```
Comment (e1himself) [inline src/guards/isUploadedImage.ts]: > `file?.original_height`

why is this necessary? Isn't `isPlainObject` already checking for `file` object to be not null?

File: src/guards/isUploadedFile.ts (hunk 1 above)
Comment (aspirisen) [reply src/guards/isUploadedFile.ts]: There is no type information that the object is not nullable and TS complains here

Comment (e1himself) [reply src/guards/isUploadedFile.ts]: It won't complain if `file` will stay typed as `any`.

Technically speaking, `file: Record<string, any> | undefined | null` type is not correct. As the type-guard checks any value. Not just `null | undefined | object`.

Comment (aspirisen) [reply src/guards/isUploadedFile.ts]: `any` is a super type, adding "any | null | undefined" is the same as just "any". Because of that we have got an issue, `typeof null` is object, and if TS would know that the argument could be null it will complain about that and forbid accessing properties

Comment (e1himself) [reply src/guards/isUploadedFile.ts]: I guess the goal of this change is to let TS help us avoid mistakes as in `isUploadedImage`:

> ```ts
> return (
>        typeof file === 'object' &&
>        typeof file.original_height === 'number' &&
> ```

Right? 

Then `file: unknown` is a better way to do so. Please see #2

**Review by e1himself — APPROVED** [review]

**Comment by e1himself** [comment]
@aspirisen feel free to merge and proceed with this one
